service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EhrExtractMapper.java

```diff
@@ -49,7 +55,7 @@ public EhrExtractTemplateParameters mapBundleToEhrFhirExtractParams(
         ehrExtractTemplateParameters.setFromOdsCode(getGpcStructuredTaskDefinition.getFromOdsCode());
         ehrExtractTemplateParameters.setAvailabilityTime(DateFormatUtil.toHl7Format(timestampService.now()));
         ehrExtractTemplateParameters.setAgentDirectory(agentDirectoryMapper.mapEHRFolderToAgentDirectory(
-            bundle, getGpcStructuredTaskDefinition.getNhsNumber()));
+            bundle, patientNhsNumber));
```

Comment:
Could call method here

Reply:
For readability it is often better to break things out into local variables. It is easy to get lost in nested methods calls within method parameters.